tokio-trace-serde/examples/serde_shaved_yak.rs

```diff
@@ -0,0 +1,136 @@
+extern crate serde_json;
+
+#[macro_use]
+extern crate tokio_trace;
+extern crate tokio_trace_core;
+extern crate tokio_trace_serde;
+
+use std::sync::atomic::AtomicUsize;
+use std::sync::atomic::Ordering;
+
+use serde_json::json;
```

Comment:
This doesn't compile on Rust 1.26.0; we need to replace it with the `#[macro_use]` attribute on the `extern crate` statement above for that CI build to pass.